ckanext/unaids/tests/test_blueprints.py

```diff
@@ -11,3 +15,69 @@ class TestValidateUserProfileBlueprint(object):
     def test_annonymous_access_to_index_page(self, app):
         index_response = app.get("/", follow_redirects=False)
         assert index_response.status_code == 200
+
+
+@pytest.fixture
+def test_org_download(app, test_organization):
+    url = toolkit.url_for(
+        'user_lists.org_member_download',
+        group_id=test_organization['name']
+    )
+    org_admin_username = test_organization['users'][0]['name']
+    return app.get(url, extra_environ={'REMOTE_USER': org_admin_username})
+
+
+@pytest.mark.ckan_config("ckan.plugins", "unaids pages")
```

Comment:
After you rebase the branch on my previous changes with job_title you'll need to add ytp_request plugin here:
```
@pytest.mark.ckan_config("ckan.plugins", "ytp_request unaids pages")
```